core/templates/dev/head/components/state-editor/state-editor-properties-services/state-editor.service.ts

```diff
@@ -36,7 +36,7 @@ oppia.factory('StateEditorService', [
     // service, the given solutions would be automatically verified for the set
     // interaction.
     var interaction = null;
-    var misconceptions = [];
+    var misconceptionsPerSkill = [];
```

Comment:
I'm confused -- shouldn't this be a dict? (Since you're doing misconceptionsPerSkill[skillId] later)

Reply:
Done.